internal/api/grpc/execution/v3alpha/query_integration_test.go

```diff
@@ -447,8 +447,11 @@ func TestServer_ListExecutions_Request(t *testing.T) {
 			args: args{
 				ctx: CTX,
 				dep: func(ctx context.Context, request *execution.ListExecutionsRequest, response *execution.ListExecutionsResponse) error {
-					cond := request.Queries[0].GetConditionQuery()
-					resp := Tester.SetExecution(ctx, t, cond.GetCondition(), []string{targetResp.GetId()}, []string{})
+					condQuery := request.Queries[0].GetInConditionsQuery()
+					require.NotNil(t, condQuery)
+					cond := condQuery.GetConditions()[0]
+					require.NotNil(t, cond)
```

Comment:
why do you check for NotNil as there are set as the argument of the test itself?

Reply:
Had some troubles with the test, leftovers from that, removed.